resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,152 @@
+"""Converting bam to bed and normalised bigwig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    Process,
+    SchedulingClass,
+    FileField,
+    StringField
+)
+
+
+class BamToBed(Process):
+    """Takes in a bam file and calculates a normalization factor in .bed format.
+    Done by sorting with samtools and transformed with bedtools.
+    """
+
+    slug = "bam-to-bed"
+    name = "Bam to bed"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Badtools bamtobed ({{alignment|sample_name}})"
+    version = "1.0.0"
+    process_type = "data:bed"
+    category = "Other"
+    entity = {
+        "type": "sample",
+        "input": "alignment",
```

Comment:
Not sure if we really need this. What does @jkokosar think?

Reply:
`"input": "alignment"` part is redundant here, as there is only one input for this process that can be a part of entity sample.